app/src/tests/Appointment.test.tsx

```diff
@@ -1,12 +1,14 @@
 import classData from 'data/clean.json';
-import { parseCalendarJSON } from 'utils/utils';
-import { getClassById } from 'components/organisms/Appointment';
+import { parseCalendarTeacher, parseCalendarCourse } from 'utils/utils';
+
+import getTeacherById from 'components/organisms/Appointment';

```

Comment:
Currently I think its importing the Appointment function from Appointment.tsx and renaming it to getTeacherById. With the exporting suggestion I made earlier you also need to surround getTeacherById with curly braces to specify that function.